web/src/pages/indicators/[slug].page.tsx

```diff
@@ -0,0 +1,221 @@
+					&nbsp;{formatDateStr(product.publishedDate)}
+				</time>
+			</>
+		) : null,
+		product.lastMajorModificationDate != product.publishedDate ? (
+			<>
+				Last updated:
+				<time dateTime={dayjs(product.lastModified).format("YYYY-MM-DD")}>
+					&nbsp;{formatDateStr(product.lastModified)}
+				</time>
+			</>
+		) : null,
+	].filter(Boolean);
+
+	return (
+		<>
+			<NextSeo
+				title={
+					product.title + " | Indicators | Standards and Indicators | NICE"
+				}
+				description={product.metaDescription}
+				additionalLinkTags={[
+					{
+						rel: "sitemap",
+						type: "application/xml",
+						href: "/indicators/sitemap.xml",
+					},
+				]}
+			/>
+
+			<Breadcrumbs>
+				<Breadcrumb to="/">Home</Breadcrumb>
+				<Breadcrumb to="/standards-and-indicators">
+					Standards and Indicators
+				</Breadcrumb>
+				<Breadcrumb to="/standards-and-indicators/indicators">
+					Indicators
+				</Breadcrumb>
+				<Breadcrumb>{product.id}</Breadcrumb>
+			</Breadcrumbs>
+
+			<PageHeader
+				heading={product.title}
+				useAltHeading
+				id="content-start"
+				metadata={metaData}
+			/>
+
+			<Grid gutter="loose">
+				{chapters ? (
+					<GridItem
+						cols={12}
+						md={4}
+						lg={3}
+						elementType="section"
+						aria-label="Chapters"
+					>
+						<PublicationsChapterMenu chapters={chapters} />
+					</GridItem>
+				) : null}
+				<GridItem cols={12} md={8} lg={9} elementType="section">
+					{product.summary ? (
+						<div
+							dangerouslySetInnerHTML={{ __html: product.summary }}
+							className={styles.summary}
+						/>
+					) : null}
+					{nextPageLink ? (
+						<PrevNext
+							nextPageLink={{
+								text: nextPageLink.title,
+								destination: nextPageLink.url,
+								elementType: ({ children, ...props }) => (
+									<Link {...props} scroll={false}>
+										{children}
+									</Link>
+								),
+							}}
+						/>
+					) : null}
+				</GridItem>
+			</Grid>
+		</>
+	);
+}
+
+export const getServerSideProps: GetServerSideProps<
+	IndicatorsDetailsPageProps
+> = async ({ params }) => {
+	if (
+		!params ||
+		!params.slug ||
+		Array.isArray(params.slug) ||
+		!params.slug.includes("-")
+	) {
+		return { notFound: true };
+	}
+
+	const productTypes = await getAllProductTypes();
+	const productType = productTypes.find((p) => p.identifierPrefix === "IND");
+
+	if (!productType) {
+		throw Error("Indicator product type could not be found");
+	}
+
+	const [id, ...rest] = params.slug.split("-");
+
+	const product = await getProductDetail(id);
+
+	if (
+		isErrorResponse(product) ||
+		product.id.toLowerCase() !== id.toLowerCase()
+	) {
+		return { notFound: true };
+	}
+
+	const titleExtractedFromSlug = rest.join("-").toLowerCase();
+
+	const slugifiedProductTitle = slugify(product.title);
+	if (titleExtractedFromSlug !== slugifiedProductTitle) {
+		const redirectUrl = "/indicators/" + id + "-" + slugifiedProductTitle;
```

Comment:
Use of the `getProductPath` util function we have here would be good. I can see a scenario where we might change the slugging to use short title, or just use id (depending on making changes in syndication) so using `getProductPath`  would mean we don't need to refactor multiple places

Reply:
swapped this out...the type it's expecting is productLite however